cryptographic_estimators/SDEstimator/SDWorkfactorModels/workfactor_helper.py

```diff
@@ -93,20 +87,27 @@ def may_ozerov_near_neighbor_time(list_size: float, vector_length: float, target
     d = inverse_binary_entropy(1 - normed_list_size)
 
     if normed_weight <= 2 * d * (1 - d):
-        mo_exp = (1 - normed_weight) * (1 -
-                                        binary_entropy((d - normed_weight / 2) / (1 - normed_weight)))
+        mo_exp = (1 - normed_weight) * (1 - binary_entropy((d - normed_weight / 2) / (1 - normed_weight)))
     else:
         mo_exp = 2 * normed_list_size + binary_entropy(normed_weight) - 1
-    return max(mo_exp * vector_length, 2 * list_size - vector_length + binomial_approximation(vector_length, target_weight))
+    return max(
+        mo_exp * vector_length,
+        2 * list_size - vector_length + binomial_approximation(vector_length, target_weight),
+    )
 
 
 def representations_asymptotic(target_weight: float, weight_to_cancel: float, vector_length: float):
-    """
-    computes the asymptotic number of representations of a length-$vector_length$ weight-$target_weight$ vector
-    via the sum of two length-$vector_length$ weight-($target_weight$/2+$weight_to_cancel$) vectors
+    """Computes the asymptotic number of representations of a length-`vector_length` weight-`target_weight` vector.
+
+    This is done via the sum of two length-`vector_length` weight-(`target_weight`/2 + `weight_to_cancel`) vectors.
```

Comment:
the meaning has changed. Please update to

Computes the asymptotic number of representations of a length-`vector_length` weight-`target_weight` vector constructed as sum of two length-`vector_length` weight-(`target_weight`/2 + `weight_to_cancel`) vectors.